.github/compatibility-test-matrices/unreleased/client.json

```diff
@@ -23,6 +25,7 @@
     "TestClientTestSuite"
   ],
   "test": [
+    "TestClientUpdateProposal_Succeeds",
     "TestRecoverClient_Succeeds",
```

Comment:
I think we will need to do something special for `TestRecoverClient_Succeeds` in this scenario, because the message will only be available for >= v8.0.x.

Reply:
same for scheduling IBC upgrade, which I think needs to be added here

Reply:
True, I added the test as well here now.